curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f280dda5040f707c76b4a.md

```diff
@@ -23,7 +23,8 @@ Use the `.repeat()` method on your `character`, and give it `i` for the number.
 You should use the `.repeat()` method.
 
 ```js
-assert.match(__helpers.removeJSComments(code), /\.repeat\(/);
+assert.match(__helpers.removeJSComments(code), /rows\.push\(character\.repeat\(\s*i\s*\)\)/);
```

Comment:
While this does solve the issue, this should realistically be in another test case.

Reply:
You're right, fixed.